frontend/src/services/bs-api/bs-api.service.ts

```diff
@@ -39,6 +39,15 @@ class BSApi {
     );
   }
 
+  public deleteSpace(id: string): Promise<boolean> {
+    return this.#http.load(
+      joinItems(this.#apiPrefix, ApiPath.BS, BSApiPath.SPACES, '/', id),
```

Comment:
```suggestion
      joinItems(this.#apiPrefix, ApiPath.BS, BSApiPath.SPACES, id),
```

Reply:
or add enum

Reply:
![image](https://user-images.githubusercontent.com/86227025/155290993-0d18ecae-2a42-46e3-b786-9ea6cd01918f.png)
`joinItems(this.#apiPrefix, ApiPath.BS, BSApiPath.SPACES, BSApiPath.ROOT, id)` ?

Reply:
then, add enum for it